pandas-stubs/core/dtypes/base.pyi

```diff
@@ -3,9 +3,6 @@ from pandas.core.arrays import ExtensionArray
 from pandas._typing import type_t
 
 class ExtensionDtype:
-    def __eq__(self, other) -> bool: ...
-    def __hash__(self) -> int: ...
-    def __ne__(self, other) -> bool: ...
     @property
     def na_value(self): ...
```

Comment:
I think we should make `na_value` return `Any`

Also make `kind` return a `Literal`, one of `biufcmMOSUV` according to docs

Reply:
Done.